WordPress/wordpress-playground
# Fix/404 error if requested path doesn't exist

Resolves: #304 

This pr adds a 404 error if the requested path doesn't exist

### Testing instructions

- Create an empty folder
- Run `wp-now` and make sure you add this folder as a `path`: ` npx nx preview wp-now start --path=<your-empty-folder>`
- Ensure that you don't see any errors in the console
- Ensure that you see `404 File not found` in the browser
- Change the URL to something else. eg: `/file-not-exist.php`.
- Ensure that you still see `404 File not found`
